src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: $border;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(120%, 120%);
+  }
+
+  &__pic {
+    background-image: $cardpic;
+    background-size: 160px 134px;
+    background-repeat: no-repeat;
+    background-position: center;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    width: 166px;
+    height: 36px;
+    font-family: $fontfamily;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    margin-top: 40px;
+    color: $maincolor;
+    transition: color 0.3s;
+  }
+
+  &__code {
+    width: auto;
+    height: auto;
+    font-family: $fontfamily;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    margin-top: 4px;
+    color: $secondarycolor;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-self: center;
+    align-items: center;
+    width: 166px;
+    height: 16px;
+    font-family: $fontfamily;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    margin-top: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    font-family: $fontfamily;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: $secondarycolor;
+
+    &-amount {
+      font-family: $fontfamily;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      color: $maincolor;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    border: none;
+    border-radius: 5px;
+    background-color: $buttoncolor;
+    padding: 0;
+    cursor: pointer;
+    font-family: $fontfamily;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    color: #fff;
+    text-transform: uppercase;
+
+    &:hover {
+      border: 1px solid $buttoncolor;
+      background-color: white;
+      color: $buttoncolor;
+    }
+  }
+}
+
+.card:hover .card__name {
+  color: $cardtexthover;
+}
```

Comment:
```suggestion
&:hover &__name {
  color: $cardtexthover;
}
```

Reply:
you can do this and move it inside the block

Reply:
> you can do this and move it inside the block

![image](https://github.com/mate-academy/layout_catalog/assets/169940235/c4a15548-6ccb-4227-a121-e9f6e5034880)

![image](https://github.com/mate-academy/layout_catalog/assets/169940235/b2d63ebc-8327-45e7-aba4-5f733eb2f8ec)


I cant understand why error appears